Core/clim-basic/clim-basic.asd

```diff
@@ -20,7 +20,12 @@
     :serial t
     :components ((:file "coordinates")
                  (:file "transforms")
-                 (:file "regions")))
+                 (:file "regions")
+                 (:file "bounding-rectangle")
+                 (:file "region-bounding-rectangles")
+                 (:file "region-transformations")
+                 (:file "region-predicates")
+                 (:file "region-composition")))
```

Comment:
Why the `region-` prefix?

Reply:
bounding-rectangles contains the bounding rectangle protocol (regions "just" implement it), same goes for transformations. This is to make it clear, that in these files this and that protocol is implemented *for regions*.

Reply:
I see.